autoscheduler/frontend/src/components/GoogleButton.tsx

```diff
@@ -0,0 +1,25 @@
+// import * as React from 'react';
```

Comment:
If I un-commented this file, would it work? If not, let's add it in a future commit. Also, if we're going to keep it, can we switch it to a block-comment rather than individually commenting every line? That makes it easier to get up and running.

Reply:
Gannon already discussed this, closing mine.